tests/test_multi_standbys.py

```diff
@@ -296,3 +296,67 @@ def test_020_start_node3_again():
     assert node1.has_needed_replication_slots()
     assert node2.has_needed_replication_slots()
     assert node3.has_needed_replication_slots()
+
+def test_021_most_advanced_standby_inaccessible():
+
+    global node1
+    global node2
+    global node3
```

Comment:
Not needed here, you don't assign anything to those variables. Python is weird sometimes.